src/VisualStudio/CSharp/Impl/Options/IntelliSenseOptionPageControl.xaml

```diff
@@ -74,27 +72,15 @@
 
                     <CheckBox x:Uid="Show_items_from_unimported_namespaces"
                                   x:Name="Show_items_from_unimported_namespaces"
-                                  Content="{x:Static local:IntelliSenseOptionPageStrings.Option_Show_items_from_unimported_namespaces}" 
-                              Checked="Show_items_from_unimported_namespaces_CheckedChanged"       
-                              Unchecked="Show_items_from_unimported_namespaces_CheckedChanged"
-                              Indeterminate="Show_items_from_unimported_namespaces_CheckedChanged"
-                              IsThreeState="True"/>
+                                  Content="{x:Static local:IntelliSenseOptionPageStrings.Option_Show_items_from_unimported_namespaces}" />
```

Comment:
`Show_items_from_unimported_namespaces`, `Tab_twice_to_insert_arguments` and `Show_new_snippet_experience` 
are using the three state checkbox tricks to indicate they are in experiment.
However, I found
1. `Show_items_from_unimported_namespaces` is on by default one. I can't find its feature flag.
2. `Tab_twice_to_insert_arguments` is off by default now. I also can't find the feature flag. I synced with Sam and he intend to keep it off.
3. `Show_new_snippet_experience` has the feature flag. But it no longer need to use the three state trick because the value could be fetched from the feature flag when it's null.